src/reformat.c

```diff
@@ -115,7 +126,8 @@ avifBool avifGetYUVColorSpaceInfo(const avifImage * image, avifYUVColorSpaceInfo
         return AVIF_FALSE;
     }
 
-    // Removing 400 here would break backward behavior but would respect the spec.
+    // Removing 400 here would break backward behavior but would respect the
+    // spec.
```

Comment:
This just looks like it got reflowed and can be removed from the change.

Reply:
Done